lib/src/participant/local.dart

```diff
@@ -200,34 +200,36 @@ class LocalParticipant extends Participant<LocalTrackPublication> {
         publishOptions.backupCodec!.codec != publishOptions.videoCodec) {
       simulcastCodecs = <lk_rtc.SimulcastCodec>[
         lk_rtc.SimulcastCodec(
-            codec: publishOptions.videoCodec,
-            cid: track.getCid(),
-            enableSimulcastLayers: true),
+          codec: publishOptions.videoCodec,
+          cid: track.getCid(),
+        ),
         lk_rtc.SimulcastCodec(
-            codec: publishOptions.backupCodec!.codec.toLowerCase(),
-            cid: '',
-            enableSimulcastLayers: publishOptions.backupCodec!.simulcast),
```

Comment:
these aren't required anymore?

Reply:
When I upgraded the protocol, enableSimulcastLayers seemed to be removed

Reply:
https://github.com/livekit/protocol/commit/13abb13b71b7461d04be5b651a9938454655bf2c

Reply:
got it. I guess we don't use it on the server side.